src/main/java/com/exadel/coolDesking/bot/HotDeskingBot.java

```diff
@@ -0,0 +1,41 @@
+package com.exadel.coolDesking.bot;
+
+import org.telegram.telegrambots.meta.api.methods.BotApiMethod;
+import org.telegram.telegrambots.meta.api.methods.updates.SetWebhook;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.starter.SpringWebhookBot;
+
+public class HotDeskingBot extends SpringWebhookBot {
+    private final String BOT_USERNAME;
+    private final String BOT_TOKEN;
+    private final String BOT_PATH;
+    private final UpdateReceiver updateReceiver;
+
+    public HotDeskingBot(SetWebhook setWebhook, UpdateReceiver updateReceiver, String BOT_TOKEN, String BOT_USERNAME, String BOT_PATH) {
```

Comment:
```suggestion
    public HotDeskingBot(SetWebhook setWebhook, UpdateReceiver updateReceiver, String token, String username, String path) {
```
Don't use uppercase for method parameters